File: example_problems/hello/submissions/accepted/extra-file-cs/test-hello.cs
/*
 * This should give CORRECT on the default problem 'hello',
 * since the random extra file will not be passed.
 *
 * @EXPECTED_RESULTS@: CORRECT
 */

using System;

public class Hello
{
	public static void Main(string[] args)
	{
#if ONLINE_JUDGE
		Console.Write("Hello world!\n");
#else
		Console.Write("ONLINE_JUDGE not defined\n");
#endif
	}
}

File: example_problems/hello/submissions/accepted/extra-file-cc/test-hello.cc
/*
 * This should give CORRECT on the default problem 'hello',
 * since the random extra file will not be passed to g++.
 *
 * @EXPECTED_RESULTS@: CORRECT
 */

#include <iostream>
#include <string>

using namespace std;

int main()
{
	string hello("Hello world!");
	cout << hello << endl;
	return 0;
}
